internal/generate/olm-catalog/descriptor/descriptor.go

```diff
@@ -1,261 +0,0 @@
-// Copyright 2019 The Operator-SDK Authors
```

Comment:
This package still needs to stick around for the new CSV bases to work. #2792 should be updated to remove these. You can just leave this package in `internal/generate/olm-catalog/descriptor` for now.